CONTRIBUTING.md

```diff
@@ -141,6 +143,7 @@ Finally, **if you're running into weird problems and nothing seems to be working
 ```bash
 cd optimism
 pnpm clean
+pnpm install
```

Comment:
do we need these extra installs that were added?

Reply:
Yep - `pnpm clean` removes `node_modules` recursively in the monorepo root.